apollo-compiler/src/main/kotlin/com/apollostack/compiler/ir/Fragment.kt

```diff
@@ -17,24 +16,34 @@ data class Fragment(
     val fragmentsReferenced: List<String>
 ) : CodeGenerator {
   /** Returns the Java interface that represents this Fragment object. */
-  override fun toTypeSpec(): TypeSpec =
-      SchemaTypeSpecBuilder().build(interfaceTypeName(), fields, fragmentSpreads, inlineFragments)
+  override fun toTypeSpec(abstract: Boolean, reservedTypeNames: List<String>,
```

Comment:
nit: `abstract` is a reserved Java keyword, maybe rename it for clarity (it also messes up the syntax highlighting)